app/Http/Controllers/PostsController.php

```diff
@@ -0,0 +1,132 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Post;
+use App\PostStatus;
+use App\User;
+use Illuminate\Http\Request;
+
+class PostsController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+
```

Comment:
Do not add extra space at the very beginning of methods.

Reply:
Removed.